integration-cli/docker_cli_run_test.go

```diff
@@ -3610,3 +3610,10 @@ func (s *DockerSuite) TestRunWrongCpusetMemsFlagValue(c *check.C) {
 	expected := "Error response from daemon: Invalid value 1-42-- for cpuset mems.\n"
 	c.Assert(out, check.Equals, expected, check.Commentf("Expected output to contain %q, got %q", expected, out))
 }
+
+func (s *DockerSuite) TestRunWithPull(c *check.C) {
+	testRequires(c, Network)
+	out, _, err := dockerCmdWithError("run", "--pull", "hello-world")
+	c.Assert(err, check.IsNil, check.Commentf(out))
+	c.Assert(out, checker.Contains, "Downloaded newer image for hello-world:latest", check.Commentf("expected to download latest image from docker hub"))
```

Comment:
@thaJeztah  you are right, my mistake here, here should be `Downloaded newer image` || `Image is up to date`